pkg/controller/kieapp/constants/constants.go

```diff
@@ -260,6 +260,23 @@ const (
 	//Envs
 	KubeNS     = "KUBERNETES_NAMESPACE"
 	KubeLabels = "KUBERNETES_LABELS"
+
+	//Kafka extension
+	KIE_SERVER_KAFKA_EXT_DISABLED          = "KIE_SERVER_KAFKA_EXT_DISABLED"
```

Comment:
Not sure if it is needed here, besides the tests, what are the other use case?

Reply:
Yep, only test, moving there